core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -151,7 +151,13 @@ protected[controller] object RestApiCommons {
  */
 protected[controller] trait RespondWithHeaders extends Directives {
   val allowOrigin = `Access-Control-Allow-Origin`.*
-  val allowHeaders = `Access-Control-Allow-Headers`("*")
+  val allowHeaders = `Access-Control-Allow-Headers`(
+    "Authorization",
+    "Origin",
+    "X-Requested-With",
+    "Content-Type",
+    "Accept",
+    "User-Agent")
```

Comment:
Should we make this list configurable?

Reply:
probably the ultimate right course of action is to make both fields (allow-origin, allow-headers) configurable.

the reason i've put in this short-term fix is that my previous PR probably broke some use cases; whereas previously web actions would work from any browser (assuming only those two previously allowed headers were passed), now web actions will only work from chrome (but will support *any* headers)

with this PR, we restore functionality on all browsers for web actions, with a somewhat expanded list of allow headers)

i would suggest that we do the right thing in a subsequent PR, but this PR i feel fills an important a short-term regression fix until we can get the right PR in place?

Reply:
SGTM

Reply:
Can we at least define the list of headers in a centralized place so we don't end up copying them around?

Reply:
sounds good. i'll work on this in a bit.